Features/FindDataById/requires.txt

```diff
@@ -0,0 +1,17 @@
+## Stores information about all the libraries, modules, and packages that are used in this project.
```

Comment:
Add the missing from the following to the file or replace the entire list:

flake8>=3.8.4
pandas>=1.2.2
pytest
py>=1.10.0
pytest-html>=3.1.1
pytest-json-report
selenium == 4.11.0
openpyxl
matplotlib >= 3.3.4
pytest-rerunfailures
pytest-xdist
pytest-xdist[psutil]
pytest-order
requests
imap-tools
beautifulsoup4
html5lib
pytest-metadata
pyotp >=2.6.0